### ISSUE TITLE
Implementation of WorldExpo'10 dataset

### ISSUE THREAD

**Issue by desenzhou** [CommissarMa/CSRNet-pytorch#4]
Hi, I'm using CSRNet for crowd counting on world Expo'10 dataset. However, I cannot reproduce the results of the paper(8.6 MAE), I only got a MAE around 10.2. Could you please provide the implementation of world Expo'10 dataset(e.g. density map generation or how to utilize the ROI and evalutation), or at least provide some implementation details( learning rate, batch size ect.). Thank you so much!!

**Comment by CommissarMa** [comment]
sorry，I didn't conduct experiments on world expo. And only do it on Shanghaitech part a. but flip the image and upsample image can enhance the performance.&nbsp; the learning rate can vary from 1e-5 to 1e-7 to finetune the model gradually. decay is default.



---Original---

**Comment by desenzhou** [comment]
OK~Thank you anyway!

**Comment by weibowei** [comment]
> Hi, I'm using CSRNet for crowd counting on world Expo'10 dataset. However, I cannot reproduce the results of the paper(8.6 MAE), I only got a MAE around 10.2. Could you please provide the implementation of world Expo'10 dataset(e.g. density map generation or how to utilize the ROI and evalutation), or at least provide some implementation details( learning rate, batch size ect.). Thank you so much!!

Hi, could you give me a download link about WorldExpo’10 Crowd Counting Dataset?
I have send emails to xierong@sjtu.edu.cn, but I didn't get any reply. 
If you have the dataset, could you send download link to me ? My email is 220191681@seu.edu.cn.
Thanks!
